apps/ledger-live-mobile/.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     "airbnb",
     "prettier",
     "plugin:json/recommended",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
Should we also add the typed rules ? https://typescript-eslint.io/docs/linting/typed-linting/